docker/build/mssql-init/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# escape=`
+
+ARG BASE_IMAGE
+ARG HEADLESS_SERVICES_IMAGE
+ARG SPE_ASSETS_IMAGE
+
+FROM ${HEADLESS_SERVICES_IMAGE} AS headless_services
+FROM ${SPE_ASSETS_IMAGE} AS spe_assets
+FROM ${BASE_IMAGE}
+
+RUN Install-PackageProvider -Name NuGet -Force; `
+    Set-PackageSource -Name PSGallery -Trusted ; `
+    Install-Module -Name SqlServer;
+
+COPY ./sql /sql
+COPY ./tools/scripts\ .\
+COPY ./data /data
```

Comment:
Should the `COPY` order be optimized for caching with:
1. Modules (Headless, SPE) first
2. sql
3. tools/scripts
4. data last

Reply:
Done